fabfile.py

```diff
@@ -212,7 +212,7 @@ def install_node10():
                 return True
 
         version = '0.10.26'
-        node_url = 'http://nodejs.org/dist/v{0}/node-v{0}-linux-arm-pi.tar.gz'
+        node_url = 'http://nodejs.org/dist/v{0}/node-v{0}.tar.gz'
```

Comment:
That one won't work for old Raspberry Pi. If the idea is to remove the Pi 1, we should mention it in the README.